jsonpatch.py

```diff
@@ -637,7 +649,8 @@ def apply(self, obj):
 
 class DiffBuilder(object):
 
-    def __init__(self):
+    def __init__(self, dumps):
```

Comment:
Could we make this an optional argument?

Reply:
Done. Defaults to `json.dumps`